tflitehub/lightning_fp16_test.py

```diff
@@ -1,4 +1,5 @@
-# RUN: %PYTHON %s
+# RUN: %PYTHON %s %config_flag
+# XFAIL: vmvx
```

Comment:
@jpienaar This test fails only with vmvx backend. Should I file a GH issue to track it?

Reply:
Yes - I'm pretty sure I know what this is from (sub-32 bit has some gaps VMVX side - its mostly in the conversions to the underlying storage and loading back, these aren't a priority for us at the moment but please flag if blockers).

Reply:
It is not a blocker for our work. I will just tag it as XFAIL for now. :)